app/assets/stylesheets/application.scss

```diff
@@ -71,6 +71,7 @@
 @import "components/cursor";
 @import "components/umm-version-label";
 @import "components/variable_generation";
+@import "components/timeline";
```

Comment:
did you say you tried the gem and then ended up doing it all by hand? Does this mean you need to remove this or you ended up using a part of it?

Reply:
I tried a prebuilt node component, but there are no remnants of it in this PR.  This is how we incorporate our css into sass.